dedibox-network/rpn/how-to/connect-rpn-san.mdx

```diff
@@ -7,7 +7,7 @@ dates:
 ---
 
 
-[RPN-SAN](https://www.scaleway.com/en/dedibox/storage/rpn-san/) is a SAN storage network solution (Storage Area Network) accessible in block mode, performance-oriented and available in iSCSI through the RPN private network of your Dedibox.
+[RPN-SAN](https://www.scaleway.com/en/dedibox/storage/rpn-san/) is a SAN storage network solution (Storage Area Network) accessible in block mode, performance-oriented and available in iSCSI through the RPN Private Network of your Dedibox.
```

Comment:
I think here it isn't a product name (only for Elements)

Reply:
```suggestion
[RPN-SAN](https://www.scaleway.com/en/dedibox/storage/rpn-san/) is a SAN storage network solution (Storage Area Network) accessible in block mode, performance-oriented and available in iSCSI through the RPN private network of your Dedibox.
```